resources/js/views/Locations/Countries.vue

```diff
@@ -61,13 +61,13 @@
                     <div class="columns">
                         <div class="column is-half is-offset-3">
                             <div class="columns">
-                                <div class="column is-one-third">
-                                    <h1 class="subtitle is-5" style="color: black; text-align: center;">
```

Comment:
Why are we removing the black here?

Reply:
We already have `style="color:black;"` on the `strong` tag.